frontend/src/components/tutorialRequestModal/TutorialRequestModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { Form, Modal, Input, Button, message, Select } from 'antd';
+import TextArea from "antd/lib/input/TextArea";
+import { useForm } from "antd/lib/form/Form";
+import { useState } from "react";
+import { createTutorialRequest } from "../../api/api";
+import { TutorialRequest } from "../../types/Tutorial";
+import { validateMessages } from "../../utils/Messages";
+
+interface Props {
+    isModalVisible: boolean,
+    setIsTutorialRequestModalVisible: (visible: boolean) => void
+}
+
+export const TutorialRequestModal: React.FC<Props> = ({ isModalVisible, setIsTutorialRequestModalVisible }) => {
+
+    const [loading, setLoading] = useState(false);
+    const [form] = useForm();
+    const { Option } = Select;
+
+    const onFinish = (values: any) => {
+        console.log(values.semester);
+        setLoading(true);
+        createTutorialRequest({
+            title: values.title,
+            semester: Number(values.semester),
+            description: values.description
+        } as TutorialRequest).then(res => {
+            setLoading(false);
+            message.success("Tutoriumsanfrage erfolgreich erstellt.");
+            setIsTutorialRequestModalVisible(false);
+            form.resetFields()
+        }, err => {
+            setLoading(false);
+            message.error("Tutoriumsanfrage konnte nicht erstellt werden.");
+        }
+
+        )
+    };
+
+    const onCancel = () => {
+        setIsTutorialRequestModalVisible(false)
+        form.resetFields()
+    }
+
+    return (
+        <Modal
+            title="Tutoriumsanfrage erstellen"
+            destroyOnClose={true}
+            visible={isModalVisible}
+            onCancel={onCancel}
+            width={600}
+            footer={[
+                <Button loading={loading} htmlType="submit" type="primary" onClick={e => form.submit()}>
```

Comment:
Würde nicht nur den Button, sondern auch alle Input-Felder in der Form mithilfe von `disabled={loading}` disablen damit man nicht nochmal was ändern kann während es lädt